Docs: EC2 Section Refactoring (#163)
C2Devel/terraform-provider-croccloud: docs/data-sources/eip.md

}
```

## Argument reference

The arguments of this data source act as filters for querying the available Elastic IPs.
The given filters must match exactly one Elastic IP whose data will be exported as attributes.

* `filter` - (Optional, [Block](#filter)) One or more name/value pairs to use as filters.
    * _Valid values:_ See supported names and values in [EC2 API documentation][describe-addresses]

> * `id` - (Optional, String) The ID of the allocation of the specific VPC Elastic IP to retrieve.

Reviewer: тут надо бы переформулировать по человечески

* `public_ip` - (Optional, String) The public IP of the specific Elastic IP to retrieve.
* `tags` - (Optional, Map of strings) Key-value pairs. Must exactly match pairs on the desired resource.

### filter

* `name` - (Required, String) The name of the filter.
    * _Constraints:_ Filter names are case-sensitive
* `values` - (Required, List of strings) One or more filter values.
    * _Constraints:_ Filter values are case-sensitive

## Attribute reference

### Supported attributes

In addition to all arguments above, the following attributes are exported:

> * `association_id` - (String) The ID representing the association of the address with an instance in a VPC.

Reviewer: ```suggestion
* `association_id` - (String) The ID of the address association with an instance in a VPC.
```

> * `domain` - (String) Indicates whether the address is for use in EC2-Classic (standard) or in a VPC (vpc).

Reviewer: ```suggestion
* `domain` - (String) Indicates whether the address is for use in EC2 Classic (standard) or in a VPC (vpc).
```

> * `id` - (String) If VPC Elastic IP, the allocation identifier.

Reviewer: переформулировать

* `instance_id` - (String) The ID of the instance that the address is associated with (if any).
* `network_interface_id` - (String) The ID of the network interface.

> * `network_interface_owner_id` - (String) The ID of the project that owns the network interface.

Reviewer: я бы здесь (и в других аналогичных местах) так переформулировал
```suggestion
* `network_interface_owner_id` - (String) The ID of the project that the network interface belongs to.
```

> * `private_ip` - (String) The private IP address associated with the Elastic IP address.

Reviewer: наверно, везде стоит добавить слово address
